packages/runtime/container-runtime/src/dataStores.ts

```diff
@@ -145,7 +145,13 @@ export class DataStores implements IDisposable {
         if (this.contexts.has(attachMessage.id)) {
             const error = new DataCorruptionError(
                 "duplicateDataStoreCreatedWithExistingId",
-                extractSafePropertiesFromMessage(message),
+                {
+                    ...extractSafePropertiesFromMessage(message),
+                    dataStoreId: {
```

Comment:
Can you create a bug to evalute whether dataStoreId qualifies as PackageData or should it be under a different tag. Then  add the link here in a comment.

Reply:
Added!